plugins/qeta-backend/src/service/routes/attachments.ts

```diff
@@ -97,6 +98,20 @@ export const attachmentsRoutes = (router: Router, options: RouterOptions) => {
     let imageBuffer: Buffer;
     if (attachment.locationType === 'database') {
       imageBuffer = attachment.binaryImage;
+    } else if (attachment.locationType === 's3') {
+      const bucket = config.getOptionalString('qeta.storage.bucket');
```

Comment:
You probably want to check that the bucket is defined at this point and throw an error if it's not. 

Also, add these new config values to the `config.d.ts`